source/posts/2018-02-08-web-gelistiricisi-is-ilani.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Web Geliştiricisi İş İlanı
+date: 2018-02-08
+author: lab2023
+tags: iş ilanı, web geliştirici, duyuru, tr
+---
+
+![web-developer-job-application-cover](../assets/images/articles/2018-02-08-web-gelistiricisi-is-ilani/web-developer-job-application-cover.jpg)
+
+## İş Tanımı
+Ekibimize katılmak üzere en az 2 yıl deneyimli tam zamanlı web geliştiricileri arıyoruz. Geliştiricilerden sadece ürün geliştirmelerini değil blog yazısı, webinar, etkinliklere konuşmacı olarak katılması ve açık kaynak projelere katkıda bulunmasını bekliyoruz.
+
+## Teknik Nitelikler
+* HTML, CSS, JavaScript v.b. web teknolojilerine hakim olmak.
+* Ruby, Python, PHP, Java gibi web geliştirmede de kullanılabilen en az bir dile hakim olmak. 
+* Rails, Django, Laravel yada Spring gibi en az bir geliştirme çatısına hakim olmak.
+* OOP, MVC, ORM, SOLID, Tasarım şablonları, TDD & BDD, Debug, Refactoring kavramlarını bilmek.
+* Agile, Scrum kültürünü bilmek.
+
+## Tercih Nedenleri
+* Kullanmakta olduğumuz Ruby, Rails, HAML, SASS, CoffeeScript v.b. teknolojileri biliyor olmak. 
+* Uzman olunmasa bile UX, UI, prototipleme konularında fikir sahibi olmak. Kötü yada kullanışsız bir ürün görünce rahatsız olmak. 
+* Ekip üyelerimizin belirli alanlarda uzmanlaşırken yeni teknolojiler öğrenmeye kapalı olmamasını bekliyoruz.
+* React, React Native, Ember, Bootstrap, Swift, Android, Node.js, Sencha Ext JS, Cordova teknolojilerinden bazılarına veya benzer popüler teknolojilere ilgi duyuyor olmanız bir avantaj olarak değerlendirilecektir.
```

Comment:
Maybe this could be better:
**_ilgi duyuyor veya biliyor olmanız bir avantaj olarak değerlendirilecektir._**